src/createBaseForm.js

```diff
@@ -167,7 +167,15 @@ function createBaseForm(option = {}, mixins = []) {
 
       getFieldDecorator(name, fieldOption) {
         const props = this.getFieldProps(name, fieldOption);
+        let decoratorInvoked = false;
         return (fieldElem) => {
+          warning(
```

Comment:
should check if is the same name

Reply:
I'm checking if a key with the same name already exists, inside `getFieldProps` method. But there is also the case if somebody assign the return value of `getFieldDecorator` to a variable and then using it twice. Hence I'm using the `decoratorInvoked` flag to see if the decorator was called more than once.